packages/core/backend/sirius-components-core/src/main/java/org/eclipse/sirius/components/core/services/DefaultImagePathService.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.core.services;
+
+import java.util.List;
+
+import org.eclipse.sirius.components.core.api.IImagePathService;
+import org.springframework.stereotype.Service;
+
+/**
+ * An IImagePathService for the default images.
+ * @author fbarbin
+ */
+@Service
+public class DefaultImagePathService implements IImagePathService {
```

Comment:
Why is this named `Default...`?
Is it because of the _Default_ image or, because _/icons/svg_ is a default path?

Reply:
Yes I did not find a better name.
CoreImagePathService ?

Reply:
IMO:
- If you want to name it _CoreImagePathService_, a "core" should be added to the Path.
- You also may keep default but add "default" to the Path.

Reply:
Ok for the third one !

Reply:
Finally I would suggest CoreImagePathService since we already use CoreImageConstants.IMAGES_ROOT_FOLDER and CoreImageConstants.DEFAULT_SVG to reference the default.svg multiple time in the sirius-web-application module

Reply:
@gcoutable what do you think ?

Reply:
`CoreImagePathService` using `CoreImageConstants` would do it.